src/oauth2/controller/active-sessions.ts

```diff
@@ -38,4 +44,11 @@ class UserActiveSessions extends Controller {
 
 }
 
+function assertNotGroup(input: Principal): input is User | App {
```

Comment:
```suggestion
function assertNoneGroup(input: Principal): input is User | App {
```

wouldn't this make more sense?

Reply:
It asserts that 'input' is _not_ a group. Input can be either a user, group or app but it's always just 1.

An alternative could have been `assertsAppOrUser` but it's a bit more clumsy to write. Or `assertsPrincipalHasOAuthSupport` but it's getting a bit wordy then :/